_episodes/04-recipe.md

```diff
@@ -181,7 +182,7 @@ Please note the following sections:
   - diagnostic section: lines 30-42
 
     The information of which diagnostic script to run with which variables. The
-    diagnostics section has some indents that are free to call.
+    diagnostics section has some indents that are free to be called.
```

Comment:
I am not 100% sure that this is the correct expression (the one that I changed it to) since I do not totally understand what was meant with this sentence...

Reply:
Maybe it means "... that are free to be called in any order"?